i18n/about/staff.i18n.js

```diff
@@ -0,0 +1,242 @@
+            },
+            {
+                organization: 'Program',
+                members: [
+                    'Winnie',
+                    'Pochun',
+                    'Petertc',
+                    'JunWei Song',
+                    'Annie Bu',
+                    'Jordan Su',
+                    'Gina',
+                    'Kyle',
+                    'Yider',
+                    'Tony',
+                    '黃坤賢',
+                    'TingYi',
+                    'Winfred',
+                    'Shihchi',
+                    'Huei Lin',
+                    'Benson',
+                    'Jason',
+                    'Kir',
+                    'Leonard',
+                    'Lance',
+                    'Shirley',
+                ],
+            },
+            {
+                organization: 'Development',
+                members: [
+                    'Matt Wang',
+                    'Kaka Lin',
+                    'Ethan Lin',
+                    'Max',
+                    'Set Mao',
+                    'Maliao',
+                    'Flynn',
+                    'Alice Chang',
+                    '泰瑋',
+                    'Josix',
+                    'Hane',
+                    'SHY',
+                    'Phil',
+                    'Benson',
+                ],
+            },
+            {
+                organization: 'Venue',
+                members: [
+                    'David Liu',
+                    '地瓜',
+                    'Yider',
+                    'Matthew',
+                    '真瑜',
+                    'Ray',
+                    'Jason',
+                    'Juihsiang',
+                    'Kevin',
+                    'Ben',
+                    'Macs',
+                    '阿嘉',
+                    'Allen',
+                    'Ching',
+                    'J魚',
+                    'Julie',
+                    'Derek',
+                    'Luke',
+                    '陳彥愷',
+                    'Tim',
+                    'liyu',
+                ],
+            },
+            {
+                organization: 'Marketing',
+                members: [
+                    'Tumi',
+                    'Gobby Chen',
+                    '四貓',
+                    '9ukei',
+                    'Derek',
+                    '品淳',
+                    '寬寬',
+                    '小玖',
+                    'Allen Wu',
+                    'Tony',
+                    'Ken',
+                    'Mary',
+                    'Cynthia Lu',
+                    'Rain',
+                ],
+            },
+            {
+                organization: 'Registration',
+                members: ['Yucheng', 'Tumi'],
+            },
+            {
+                organization: 'Recruitment',
+                members: ['Rex', 'Josix'],
+            },
+            {
+                organization: 'Design',
+                members: ['Ben', '小玖'],
+            },
```

Comment:
update for design team
```suggestion
            {
                organization: 'Design',
                members: [
                    'Ben',
                    '小玖',
                    '王禹翔',
                    '安安',
                    'Sandy',
                    'Carol Cho',
                ],
            },
```

Reply:
Thank you. I fix them (f1f92d9).